### PR TITLE
Add Kubedoom community pack

### PR DESCRIPTION
**Pull request by lennessyy**
This PR creates a community pack that lets users to play the video game Doom (1993) in a Kubernetes cluster on Edge. This is a fun pack that extends the internet phenomenon of running DOOM on anything.

### CODE REVIEW

File: packs/kubedoom-1.0.0/pack.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "addonType": "app services",
+  "annotations": {
+    "source": "community",
+    "contributor": "spectrocloud"
+  },
+  "cloudTypes": ["all"],
```
Comment (karl-cardenas-coding) [inline packs/kubedoom-1.0.0/pack.json:7]: @lennessyy should we just make this for edge versus all? Or, you can improve the logic and provide a present for edge and one for not edge. The non-edge preset could deploy a load balancer.

Your take but if you only want this for edge then it's best make it edge only.

File: packs/kubedoom-1.0.0/README.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+- A VNC viewer to access the application. 
+
+## Parameters
+
+The following parameters are applied to the **kubedoom.yaml** manifest through the **values.yaml** file. Users do not need to take any additional actions regarding these parameters.
+
+| **Parameter**                     | **Description**                                                                | **Default Value**                           | **Required** |
+| --------------------------------- | ------------------------------------------------------------------------------ | ------------------------------------------- | ------------ |
+| `manifests.namespace`             | The namespace in which the application will be deployed.                       | `kubedoom`                            | No           |
```
Comment (karl-cardenas-coding) [inline packs/kubedoom-1.0.0/README.md:23]: The schema has this as required.

File: packs/kubedoom-1.0.0/README.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+| `manifests.images.kubedoom` | The application image that will be utilized to create the containers.          | `ghcr.io/spectrocloud/kubedoom:1.0.0` | No           |
+| `manifests.port`                  | The cluster port number on which the service will listen for incoming traffic. | `30059`                                      | No           |
+
```
Comment (karl-cardenas-coding) [inline packs/kubedoom-1.0.0/README.md:26]: All these variables are required but they have good default values.

File: packs/kubedoom-1.0.0/README.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
+# Kubedoom
+
+Kubedoom is a containerized version of the 1993 video game Doom developed by id Software. The application runs in a container that simulates a virtual desktop and needs to be accessed with a Virtual Network Computing (VNC) viewer. 
+
```
Comment (karl-cardenas-coding) [inline packs/kubedoom-1.0.0/README.md:4]: Let's mention the upstream and add a link so people can review the source code.

**Review by karl-cardenas-coding — DISMISSED** [review]
Approved with some comments

File: packs/kubedoom-1.0.0/manifests/kubedoom.yaml [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+    metadata:
+      labels:
+        app: kubedoom
+    spec:
+      serviceAccountName: kubedoom
+      containers:
+        - image: {{ .Values.images.kubedoom }}
+          env:
+            - name: NAMESPACE
+              value: default
```
Comment (vishwanaths) [inline packs/kubedoom-1.0.0/manifests/kubedoom.yaml:48]: should this also refer to  {{ .Values.namespace }} ?

**Review by vishwanaths — CHANGES_REQUESTED** [review]

File: packs/kubedoom-1.0.0/manifests/kubedoom.yaml (hunk 5 above)
Comment (lennessyy) [reply packs/kubedoom-1.0.0/manifests/kubedoom.yaml]: The original manifest was like like this and it did not affect dyployment. but yeah i think it is supposed to be kubedoom

**Comment by vishwanaths** [comment]
Closing this after discussion, as this is GNUPL.

**Pull request closed without merging**